static/css/devhub/listing.less

```diff
@@ -166,3 +169,7 @@
 #addon-delete-listing.item:hover a.delete-addon {
     color: #fff;
 }
+
+.item:hover button.link {
```

Comment:
Do we need to change the color here?

Reply:
it is similar to https://github.com/mozilla/addons-server/blob/master/static/css/devhub/listing.less#L163:L165 . The color of resume button which remains greyed on hovering on item div should turn to blue so that it looks active. The button link has to be underlined on hovering over the button for which I've added the code above.

Reply:
@eviljeff  please tell if it explains it all.

Reply:
👍